cves/kernel/CVE-2020-36558.yml

```diff
@@ -362,8 +410,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: The developer forgot to have a null pointer check AFTER obtaining a lock(had one before it).
```

Comment:
I don't like the wording, "The developer forgot".

Reply:
changed